program_v2/tests/suite-resolve/resolve.ts

```diff
@@ -179,22 +194,283 @@ describe('sol-did resolve and migrate operations', () => {
     );
   });
 
-  it('can successfully update the state of a DID', async () => {
+  it('can update the verificationMethods of a Did', async () => {
+    const existing = await service.getDidAccount();
+    expect(existing).to.not.be.equal(null);
+    let vms = [
```

Comment:
can you move the generation of payloads outside of the testfile? e.g. into `test/utils/utils.ts`